text/processing.py

```diff
@@ -256,3 +263,28 @@ def run(self):
             file_set2 = set(fin.read().split("\n"))
         with util.uopen(self.out_file, "wt") as fout:
             fout.write("\n".join(sorted(file_set1.difference(file_set2))))
+
+
+class WriteToTextFileJob(Job):
+    """
+    Write a given content into a text file, one entry per line
+    """
+
+    def __init__(
+        self,
+        content,
+    ):
+        """
+        :param list|str content: input which will be written into a text file
+        """
+        self.content = content
+
+        self.out_file = self.output_path("file.txt")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
+
+    def run(self):
+        with open(self.out_file.get_path(), "w") as f:
+            for line in self.content:
```

Comment:
I just noticed this breaks for type `str`, because then each character is written to a line, so the for loop should only be executed when it is of type list.